### PR TITLE
Add Story modals

### PR DESCRIPTION
**Pull request by ellsclytn**
I synced with master, so ignore the extra commits (the first commit is all you need).

- Styles images in structured text
- Styles story content within modals

### CODE REVIEW

File: source/layouts/Stories/index.js [hunk 1]
```diff
@@ -17,7 +17,12 @@ export default ({
       decorators={[]}
     >
       {stories.map((story, i) => (
-        <CarouselSlide key={i} link={{ href: story.image.url }} className={css(styles.slide)}>
+        <CarouselSlide key={i} className={css(styles.slide)} link={{ to: {
```
Comment (lukebrooker) [inline source/layouts/Stories/index.js]: This CarouselSlide should really be a complete reusable component. And then the image and link should be links themselves. links really shouldn't be spans and this particular one shouldn't be a button either it should be an `a`.

File: source/layouts/Story/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          margin: 'auto',
+          padding: 0,
+          width: '100%',
+          maxWidth: rhythm(25)
+        }}
+      >
+        <i className={`fa fa-times-circle ${css(styles.modalClose)}`} aria-hidden='true' onClick={() => router.push('/')} />
+        <img src={selectedStory.image.url} alt={selectedStory.image.alt} />
+        <div className={css(styles.story)}>
+          <span
```
Comment (lukebrooker) [inline source/layouts/Story/index.js]: We need to be more semantics with our html. This should be a `h3` or similar.

Comment (ellsclytn) [inline source/layouts/Story/index.js]: Bear in mind this is setting the inner HTML. Which in this case, _is_ a heading.

File: source/layouts/Story/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import css from '../../lib/css/'
+import styles from './styles'
+import {
+  rhythm
+} from '../../lib/traits'
+
+class Story extends React.Component {
+  render () {
+    const {
+			story,
```
Comment (bradparker) [inline source/layouts/Story/index.js]: Tab settings gone a little awry?

File: source/layouts/Stories/index.js [hunk 4]
```diff
@@ -16,29 +16,39 @@ export default ({
...
+              className={css(styles.thumbnail)}
+              style={{ backgroundImage: `url('${story.image.url}')` }}
+              to={storyLink}
             />
-            <span className={css(styles.openStory)}>Read more</span>
-          </div>
-        </CarouselSlide>
-      ))}
+            <div className={css(styles.text)}>
+              <h1 className={css(styles.title)}>{story.shortTitle}</h1>
```
Comment (lukebrooker) [inline source/layouts/Stories/index.js]: `h3`?

Comment (ellsclytn) [inline source/layouts/Stories/index.js]: Oops.

**Pull request merged**